### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by novalis**

### CODE REVIEW

File: node/lib/util/text_util.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+/**
+ * Indent a single string
+ * @param {String} str
+ * @param {Integer} count number of spaces to indent (default 4)
+ * @return {String} The string, indented
+ */
+exports.indent = function(str, count) {
+    if (undefined === count) {
```
Comment (bpeabody) [inline node/lib/util/text_util.js]: I'd assert on the types of `str` and `count`, and that `0 <= count`

**Comment by novalis** [comment]
OK, I just added one more, and I'll fix the others.  I'm noticing these as I work on rm, of course).

**Comment by novalis** [comment]
Actually, on reflection, don't approve -- js map is stupid.

**Comment by novalis** [comment]
Do you want to re-approve, or should I just merge?

**Pull request merged**